hudi-hadoop-common/src/test/java/org/apache/hudi/common/table/TestTimelineUtilsBackComp.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hudi.common.table;
+
+import org.apache.hudi.common.model.HoodieTableType;
+import org.apache.hudi.common.table.timeline.HoodieTimeline;
+import org.apache.hudi.common.table.timeline.TimelineUtils.HollowCommitHandling;
+import org.apache.hudi.common.table.timeline.versioning.TimelineLayoutVersion;
+import org.apache.hudi.common.testutils.HoodieCommonTestHarness;
+import org.apache.hudi.common.testutils.HoodieTestTable;
+import org.apache.hudi.common.testutils.MockHoodieTimeline;
+import org.apache.hudi.exception.HoodieException;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import java.util.stream.Stream;
+
+import static org.apache.hudi.common.table.timeline.TimelineUtils.handleHollowCommitIfNeeded;
+import static org.apache.hudi.common.testutils.HoodieTestUtils.getDefaultStorageConf;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.fail;
+
+public class TestTimelineUtilsBackComp extends HoodieCommonTestHarness {
+  @BeforeEach
+  public void setUp() throws Exception {
+    initPath();
+  }
+
+  @ParameterizedTest
+  @CsvSource({
+      "FAIL, SIX, VERSION_1",
+      "FAIL, EIGHT, VERSION_2",
+      "BLOCK, SIX, VERSION_1",
+      "BLOCK, EIGHT, VERSION_2",
+      "USE_TRANSITION_TIME, SIX, VERSION_1",
+      "USE_TRANSITION_TIME, EIGHT, VERSION_2"
```

Comment:
timeline layout versions are tied to table version. 6 is for 1 and 8 is to 2. So, why make 2 args. we could just make args as 
```
HollowCommitHandling handlingMode,
                             HoodieTableVersion tableVersion
```